### DISCUSSION TITLE
Load only PDB types option and more

### DISCUSSION THREAD

**Discussion by uniquadev** [Vector35/binaryninja-api#5303] [Ideas] [answered]
Currently, when importing debug information with a pdb file, there is no pop-up window that allows you to toggle loading options. In my scenario, I have a pdb file that doesn't match the current binary; however, the types do match. It would be useful to have the ability to toggle some loading options..

**Comment by CouleeApps** [comment, marked as answer]
I've opened #5373 to track adding this
